main/sw/inc/shellio.hxx

```diff
@@ -367,32 +363,32 @@ public:
     void   SetBaseURL( const String& rURL );
 
 	sal_Bool   IsOld() const;
-	sal_uLong  ConvertToNew();				// Textbausteine konvertieren
+	sal_uLong  ConvertToNew();				// convert AutoText entries (ger: Textbausteine)
 
-	sal_uInt16 GetCount() const;						// Anzahl Textbausteine ermitteln
-	sal_uInt16 GetIndex( const String& ) const;			// Index fuer Kurznamen ermitteln
+	sal_uInt16 GetCount() const;						// return amount of AutoText entries
+	sal_uInt16 GetIndex( const String& ) const;			// return Index of shortnames
 	sal_uInt16 GetLongIndex( const String& ) const;		//Index fuer Langnamen ermitteln
-	const  String& GetShortName( sal_uInt16 ) const; 	// Kurzname fuer Index zurueck
-	const  String& GetLongName( sal_uInt16 ) const;  	// Langname fuer Index zurueck
+	const  String& GetShortName( sal_uInt16 ) const; 	// return a shortname of an index
+	const  String& GetLongName( sal_uInt16 ) const;  	// return longname of an index
 
-	sal_Bool   Delete( sal_uInt16 );			// Loeschen
-	sal_uInt16 Rename( sal_uInt16, const String*, const String* ); // Umbenennen
+	sal_Bool   Delete( sal_uInt16 );			// delete
+	sal_uInt16 Rename( sal_uInt16, const String*, const String* ); // rename
 	sal_uLong  CopyBlock( SwTextBlocks& rSource, String& rSrcShort,
-									const String& rLong ); // Block kopieren
+									const String& rLong ); // copy block
 
-	sal_Bool   BeginGetDoc( sal_uInt16 );			// Textbaustein einlesen
-	void   EndGetDoc();						// Textbaustein wieder loslassen
+	sal_Bool   BeginGetDoc( sal_uInt16 );			// read AutoText entry
+	void   EndGetDoc();						// release AutoText entry
 
-	sal_Bool   BeginPutDoc( const String&, const String& ); // Speichern Beginn
-	sal_uInt16 PutDoc(); 								// Speichern Ende
+	sal_Bool   BeginPutDoc( const String&, const String& ); // start save
+	sal_uInt16 PutDoc(); 								// End save
 
-	sal_uInt16 PutText( const String&, const String&, const String& ); // Speichern( Kurzn., Text)
+	sal_uInt16 PutText( const String&, const String&, const String& ); // save (short., Text)
```

Comment:
IMHO "short." here is not clearly identified as "shortname". Maybe it's worth not abbreviating it.